src/PrestaShopBundle/Resources/config/services/adapter/cart.yml

```
@@ -95,3 +95,22 @@ services:
     arguments:
       - '@prestashop.adapter.command_handler.add_customization_handler'
       - '@prestashop.adapter.command_handler.update_product_quantity_in_cart_handler'
+
+  prestashop.adapter.cart.command_handler.delete_cart_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\Cart\CommandHandler\DeleteCartHandler'
```

```suggestion
  PrestaShop\PrestaShop\Adapter\Cart\CommandHandler\DeleteCartHandler:
```
We favor the FQCN notation now, even for private services and handlers